src/components/sidebar/left-sidebar/CountryList.tsx

```diff
@@ -8,8 +8,9 @@ import Translate from "../../../utils/translate/translateService";
 
 export default function CountryList() {
   const [state] = useContext(AppContext);
-
-  const aggregatedData = _.sortBy(getAggregateData(state.filtered_records, AggregationFactor.country), ['seroprevalence']).reverse();
+  // Factor in "include_in_n" for population unfiltered geography estimates
+  const must_include_in_n = state.filters.population_group.size === 0;

```

Comment:
The include_in_n criteria is only taken into account for "geography" estimates when there is no population_group filter supplied. This means that the conditions for include_in_n are different between the bar chart and the map (since for the bar chart we can have estimates for things other than geography).

For this reason, I think it's easier for the `getAggregateData` function to take in an 'include_in_n' parameter and every time we call getAggregateData from a component, it will be responsible for determining whether or not we need to `include_in_n`. It might also be easier to make it this way in case how we treat include_in_n changes eventually